.vib/spring-cloud-dataflow/goss/goss.yaml

```diff
@@ -0,0 +1,28 @@
+file:
+  /opt/bitnami/spring-cloud-dataflow/conf/application.yml:
+    exists: true
+    filetype: symlink
+    mode: "0777"
+    owner: root
+  /bitnami/spring-cloud-dataflow:
+    exists: true
+    filetype: directory
+    mode: "0775"
+    owner: root  
+  /.m2/repository/org/springframework/cloud/stream/app:
+    exists: true
+    filetype: directory
+    mode: "0644"
+    owner: root 
+command:
+  user-id-test:
+    exec: id -u
+    exit-status: 0
+    stdout:
+      - 1001
+    stderr: []
+  java-test:
+    exec: java -version
```

Comment:
I understand this test just want to check java is installed and part of `$PATH` right? Do we want want to check for a specific java version?

Reply:
No, we are avoiding checking the version in order to facilitate maintenance. 
I wanted to check that Java is present (If the command executes properly).